### PR TITLE
feat: Add endpoint for parsed clinvar copy number gain/loss

### LINKED ISSUE #285: Support ClinVar Copy Number Gain/Loss
@larrybabb :

> I like @theferrit32 ’s idea of setting up the endpoint to allow us to send the parsed components `assembly`, `chr`,` start range/number`, `end range/number`, `total copies`.  It would also be nice to have the option to send either `assembly + chr` vs `sequence accession` since both of these practices are quite common.  The assembly choices should be enumerated to `GRCh38`, `GRCh37`, `NCBI36`, `hg20`, `hg19`, `hg18`. IMO.

[Examples of ClinVar Copy Number Gain/Loss](https://www.ncbi.nlm.nih.gov/clinvar?term=((%22copy%20number%20gain%22%5BType%20of%20variation%5D%20OR%20%22copy%20number%20loss%22%5BType%20of%20variation%5D )

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #285 

Notes:
- Didn't know what to put for the text variation definition. I just used params as the label. @larrybabb is this ok?

### CODE REVIEW

**Comment by larrybabb** [comment]
@korikuzma 
```
 ParsedToAbsCnvService:
    """Given parsed ClinVar Copy Number Gain/Loss components, return Absolute
    Copy Number Variation
    :param int start: Start position as residue coordinate
    :param int end: End position as residue coordinate
    :param int total_copies: Total copies for Absolute Copy Number variation object
    :param Optional[ClinVarAssembly] assembly: Assembly. If `accession` is set,
        will ignore `assembly` and `chr`. If `accession` not set, must provide
        both `assembly` and `chr`.
    :param Optional[str] chr: Chromosome. Must set when `assembly` is set.
    :param Optional[str] accession: Accession.If `accession` is set,
        will ignore `assembly` and `chr`. If `accession` not set, must provide
        both `assembly` and `chr`.
    :return: Tuple containing Absolute Copy Number variation and list of warnings
    """
```
Wouldn't it make more sense to use `start`, `end`, `copies`, `assembly`, `chromosome`, `accession` instead of `param` for each arg, respectively?

**Comment by korikuzma** [comment]
> @korikuzma
> 
> ```
>  ParsedToAbsCnvService:
>     """Given parsed ClinVar Copy Number Gain/Loss components, return Absolute
>     Copy Number Variation
>     :param int start: Start position as residue coordinate
>     :param int end: End position as residue coordinate
>     :param int total_copies: Total copies for Absolute Copy Number variation object
>     :param Optional[ClinVarAssembly] assembly: Assembly. If `accession` is set,
>         will ignore `assembly` and `chr`. If `accession` not set, must provide
>         both `assembly` and `chr`.
>     :param Optional[str] chr: Chromosome. Must set when `assembly` is set.
>     :param Optional[str] accession: Accession.If `accession` is set,
>         will ignore `assembly` and `chr`. If `accession` not set, must provide
>         both `assembly` and `chr`.
>     :return: Tuple containing Absolute Copy Number variation and list of warnings
>     """
> ```
> 
> Wouldn't it make more sense to use `start`, `end`, `copies`, `assembly`, `chromosome`, `accession` instead of `param` for each arg, respectively?

The `:param` is docstrings. [These](https://github.com/cancervariants/variation-normalization/blob/issue-285/variation/main.py#L528-L533) are the parameters I have for the endpoint.

File: variation/main.py [hunk 1]
```diff
@@ -502,3 +504,59 @@ async def hgvs_to_relative_copy_number(
...
+    Copy Number Variation
+
+    :param int start: Start position as residue coordinate
+    :param int end: End position as residue coordinate
+    :param int total_copies: Total copies for Absolute Copy Number variation object
+    :param Optional[ClinVarAssembly] assembly: Assembly. If `accession` is set,
+        will ignore `assembly` and `chr`. If `accession` not set, must provide
+        both `assembly` and `chr`.
+    :param Optional[str] chr: Chromosome. Must set when `assembly` is set.
+    :param Optional[str] accession: Accession.If `accession` is set,
```
Comment (jsstevenson) [inline variation/main.py:545]: `Accession. If`

**Review by jsstevenson — CHANGES_REQUESTED** [review]
typo

**Review by jsstevenson — APPROVED** [review]
typo free!

**Pull request merged**